### ISSUE TITLE
IP ban causes

### ISSUE THREAD

**Issue by LeoAchilles** [finndev/PokeBuddy#823]
May I know what are the causes that would lead us to the IP ban?

And, would it be banned permanently?

**Comment by DatFIXER** [comment]
no it's just a kind of softban
I would say the new eggs feature causes issues, as using more than one account on the same IP, so just don't do that and you may be fine

**Comment by LeoAchilles** [comment]
For example, if I am using WiFi at home, would I be banned  when my IP address is not conform with the location in game?

**Comment by DatFIXER** [comment]
you could be soft banned if you jump... unless you do that, or you bot in your city, you will not have issues.

**Comment by Epic81** [comment]
@DatFIXER what vpn service do you use for win7?

**Comment by DatFIXER** [comment]
@Epic81 i don't use any vpn or ip changers, i just run one account and i'm fine

**Comment by Deuxgod** [comment]
Ok, what I think it's happening. I'm living in Switzerland and was botting in New York so I got a IP ban with incubator on. After IP ban lift I tryed boting in New York again, with incubator off, and few minutes later I was IP banned. Again, after IP ban lift, I moved to Basel in Switzerland with incubator off and botted for 2 hours without a ban. Turned on Incubator and boted all night no problem. Moved the bot to Zurich with Incubator on and I have been botting for 3 hours now without no problem.

My best guess is that the servers match your IP with the area you're botting, if it doesn't match it issues an IP ban, if it matches then you can bot away.

Once again, this is only my best guess based on my own testing. It has nothing to do with the bot.

**Comment by DatFIXER** [comment]
Sorry to destroy you theory bro; i'm currently botting in NYC and i'm not getting any bans, even if i live in Italy

**Comment by m0r73n** [comment]
And I can use the emulator in NOX i NYC without problems (living in Europe) - bot when I use this bot with the same GPS-coordinates I get IP-banned after a short while.

**Comment by Deuxgod** [comment]
Well as I said, it was just a best guess thingy. Still find it rather weird that I was getting constant IP ban when boting outside Switzerland and no problems when boting near by my area. I'll leave it like that for a while till new data comes up.

**Comment by Epic81** [comment]
Your not getting banned.The servers are garbage and are constantly crashing. 
here is a site to check the status of the servers.
when they are green and stable bots run perfect.
however its either a google login error or the u.s server in my case is down.
when both the google login server and u.s server are green and up i get a flawless running bot.
however when the server says its down specially the login server i get the no pokestops or pokemo error.
I have my account on my phone, tablet, computer and other devices. each devices has its own unique i.p address so it has nothing to do with i.p bans
http://www.mmoserverstatus.com/pokemon_go

**Comment by Deuxgod** [comment]
It has been more then proved that Niantic is issuing IP bans, what the hell are you talking about? So explain why if I log the bot on a banned IP I see no pokestops/pokemons and if I immediatly connect through a VPN it shows them? Or if people restart the router and get a new IP they can see them again also?

**Comment by Epic81** [comment]
Like i said when the site says the servers are green and up all my bots work flawlessly.
I have several accounts running under different and seperate networks with different i.p's
I have a wifi card with packet injection so i have my neighbors wifi as well as mine.
I have my main account on my wifi and a separate on his.
I have never spoofed out the country or a ridiculous distance.
I go from my house to the nearest beach and thats it.
I believe its a server error

**Comment by Epic81** [comment]
@Deuxgod  I made a fresh account and started on a wifi connection i have never used and still got no pokestops nor gyms and i havent even ran the bot just ran the account on nox player?

**Comment by Deuxgod** [comment]
That Wifi has probably his IP banned. That's the only explanation I can see for it.

**Comment by Epic81** [comment]
I will start several new account from scratch each from within the new network and i.p it receives and get back to you with the results.

**Comment by DatFIXER** [comment]
Remember that ppl in your city could report you for cheating opening a ticket, if it's that the case, man, you can do nothing.

**Comment by Epic81** [comment]
Ok so update. I loaded my primary account onto my phone which uses my carrier network.
I got all the pokestops and gyms back.
I then exited and logged into the same account on my home network via the nox emulator.
I am taking gyms as we speak.

**Comment by Epic81** [comment]
All my accounts work on bluestacks and nox player as well as with pokesniper. I only get a problem or "soft ban" with the bots

**Comment by DatFIXER** [comment]
I get problems only on Pokesniper XD!
Anyway i'm wondering how Pokesniper can mask your IP or the jumps you costantly do.

**Comment by Epic81** [comment]
@DatFIXER yeah they did an excellent job, im constantly sniping without any problems.
what problems are you having?

**Comment by DatFIXER** [comment]
@Epic81 opened before, he keeps doing like i'm soft-banned, probably i fucked up something and did a jump opening that

Tried now and update: he lemme get to coords, throw berry etc, but never catches (i'm open to suggestions bro)

Still wondering tho, how can he mask those jump?

**Comment by Epic81** [comment]
@DatFIXER  no problem i got you.
Go to pokesniper folder
find user
right click
open with notepad, or notepad++ 
input your google or ptc info
next change your latitude and longitutitude default values to where you are.
I use this site http://www.gps-coordinates.net/
gives you coords and even correct altitude.
after save the text file.
inside the pokesniper folder click on registerurl
you should now be able to click on the links directly from pokesnipers.com
you should always have your bot stopped and your app not running on your phone.
i always click on grimers 2 or 3 because the few first snipes always fail. 
after choose the ones you want and enjoy.
hope this helps.

**Comment by DatFIXER** [comment]
I'll try when i get home (imat work right now).
So i've to set coords where i live right? or where i want to catch the pokemon?

For now, many thanks for the explanation :)!

**Comment by r0nalxd** [comment]
Refering this issue to another on the same topic started earlier, #707

**Comment by Epic81** [comment]
You set the coords to where you live or where your currently catching pokemon.
these coords are found inside the user.xml file
edit the defaul latitude and longitude to your city.

the bot automatically does the rest for you when you hit the links.
